app/Console/Commands/ModdingRankCommand.php

```diff
@@ -25,60 +25,81 @@ class ModdingRankCommand extends Command
      */
     protected $description = 'Rank maps in queue.';
 
+    private bool $countOnly = false;
+    private bool $noWait = false;
+
+    public static function getStats(Ruleset $ruleset)
+    {
+        $rankedTodayCount = Beatmapset::ranked()
+            ->withoutTrashed()
+            ->withModesForRanking($ruleset->value)
+            ->where('approved_date', '>=', now()->subDays())
+            ->count();
+
+        return [
+            'availableQuota' => config('osu.beatmapset.rank_per_day') - $rankedTodayCount,
+            'inQueue' => Beatmapset::toBeRanked($ruleset)->count(),
+            'rankedToday' => $rankedTodayCount,
+        ];
+    }
+
     /**
      * Execute the console command.
      *
      * @return mixed
      */
     public function handle()
     {
-        $this->info('Ranking beatmapsets...');
+        $this->countOnly = get_bool($this->option('count-only'));
+        $this->noWait = get_bool($this->option('no-wait'));
 
-        $modeInts = array_values(Beatmap::MODES);
+        if ($this->countOnly) {
+            $this->info('Number of beatmapsets in queue:');
+        } else {
+            $this->info('Ranking beatmapsets...');
+        }
+
+        $rulesets = Ruleset::cases();
 
-        shuffle($modeInts);
+        shuffle($rulesets);
 
-        foreach ($modeInts as $modeInt) {
+        foreach ($rulesets as $ruleset) {
             $this->waitRandom();
-            $this->rankAll($modeInt);
+
+            if ($this->countOnly) {
+                $stats = static::getStats($ruleset);
+                $this->info($ruleset->name);
+                foreach ($stats as $key => $value) {
+                    $this->line("{$key}: {$value}");
```

Comment:
would have liked to use `twoColumnDetail` but didn't find any option to make the columns narrower 👀